request_id_middleware.go

```diff
@@ -11,7 +12,11 @@ func RequestIDMiddleware(next HandlerFunc) HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request, vars map[string]string) error {
 		id := r.Header.Get("X-Request-ID")
 		if len(id) == 0 {
-			id = uuid.NewV4().String()
+			uuid, err := uuid.NewV4()
+			if err != nil {
+				return fmt.Errorf("fail to generate UUID for X-Request-ID: %v", err)
```

Comment:
We can use `errgo` to wrap this error as we already have such dependency in this project

Reply:
Done, I've removed `errgo` completely